### ISSUE TITLE
About target_weight

### ISSUE THREAD

**Issue by 738654805** [ilovepose/fast-human-pose-estimation.pytorch#5]
if self.use_target_weight:
                loss += 0.5 * self.criterion(
                    heatmap_pred.mul(target_weight[:, idx]),
                    heatmap_gt.mul(target_weight[:, idx])
                )
            else:
                loss += 0.5 * self.criterion(heatmap_pred, heatmap_gt)

Q1,target_weight is introduced to keep balance between teacher and student?
Q2,loss += 0.5 * self.criterion, why the loss * 0.5?

**Comment by HuAndrew** [comment]
Just for simpler calculations.
